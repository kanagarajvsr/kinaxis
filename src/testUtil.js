
import { BrowserRouter } from 'react-router-dom'
import { render,screen,fireEvent } from '@testing-library/react';
 
export const renderWithRouter = (ui, {route = '/'} = {}) => {
    window.history.pushState({}, 'Test page', route);
    const renders = render(ui, {wrapper: BrowserRouter})
    return { 
        screen,
        fireEvent,
        ...renders
    }
  }