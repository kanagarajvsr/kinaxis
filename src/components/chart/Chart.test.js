import Chart from './Chart';
import { cleanup } from '@testing-library/react';
import {renderWithRouter} from '../../testUtil';

describe('Chart render', () => {
  afterEach(cleanup);
  const {asFragment} =  renderWithRouter(<Chart />) 
    
  test('Update Snapshot', () => { 
    expect(asFragment()).toMatchSnapshot()
  })

});
   


