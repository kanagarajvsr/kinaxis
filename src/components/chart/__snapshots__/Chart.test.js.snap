// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart render Update Snapshot 1`] = `
<DocumentFragment>
  <div
    class="recharts-responsive-container"
    style="width: 100%; height: 100%;"
  />
</DocumentFragment>
`;
